### PR TITLE
PT-2712: Phenotype Selector Bug when using Browse Related Terms

### PR DESCRIPTION
**Pull request by sdumitriu**
Fixed

### CODE REVIEW

File: components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml [hunk 1]
```diff
@@ -1325,8 +1325,7 @@ document.observe('xwiki:dom:loaded', function()
             } else if (selected == 'yes') {
               var element = event.findElement('.entry');
               if (!element.down('.descendents') || element.hasClassName('collapsed')) {
```
Comment (marta-) [inline components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml]: @sdumitriu As per my earlier comment about parents being expanded when we click 'Y' and producing a possibly confusing tree, this like could be changed to
`if ((!element.down('.descendents') || element.hasClassName('collapsed')) && element.down('.expand-tool')) {`

**Comment by sashaandjic** [comment]
When I go to see related terms and press "Y" for the parent, my original term gets listed twice. 
To reproduce: 
- type "slow" in the search and look for related terms for "Slow-growing hair",
- select "Y" for the very first term listed "Abnormality of hair growth rate", 
- notice that another "Slow-growing hair" appears

**Comment by marta-** [comment]
@sashaandjic Yep, that's exactly what I mentioned above. Your comment is a confirmation that it may indeed bother/confuse users. The proposed change should fix it. Thanks!

**Comment by sdumitriu** [comment]
@marta- @sashaandjic Applied the suggestion, please retest.

**Pull request merged**